command_server/command_server.py

```diff
@@ -1,12 +1,13 @@
 import asyncio
-import aiohttp
 import configparser
 import logging
+import aiohttp
 
-from aiohttp import server
+from aiohttp import server, web
 from aiohttp.multidict import MultiDict
-from routes import ROUTES
+from generic import routes
 from urllib.parse import urlparse, parse_qsl
+import views
 
 
 class HttpRequestHandler(aiohttp.server.ServerHttpProtocol):
```

Comment:
@ntodosiychuk  please see commet above
```
HttpRequestHandler haven't good router, and it not very common: actually it can't be applicable for using Websokets and Streams, and large responses, because it finalize response by response.write_eof(). So I decide move this logic for specific views
```